core/src/main/kotlin/io/customer/sdk/data/store/PreferenceStore.kt

```diff
@@ -0,0 +1,33 @@
+package io.customer.sdk.data.store
+
+import android.content.Context
+import android.content.SharedPreferences
+
+/**
+ * Base preference repository that can be reused among different preference repositories.
+ */
+internal abstract class PreferenceStore(val context: Context) {
```

Comment:
same as [`BasePreferenceRepository` in tracking](https://github.com/customerio/customerio-android/blob/c8cbb12eea9502a2ad8be4560e365d194124f59d/sdk/src/main/java/io/customer/sdk/repository/preference/BasePreferenceRepository.kt).